runner/ansible/README.md

```diff
@@ -4,7 +4,12 @@ Place to store Ansible playbooks that are executed by the Trento runner
 
 ## Implementing a check
 
-To be defined...
+# Table of contents
+
+- [Structure]
+   - [Metadata files](#metadata-files)
```

Comment:
Let's put other entries here:
- `Check structure`. In this sub chapter explain where the check folder should go `roles/checks/etc`. How we should select a name (explaining that this will determine the order which the test is executed), and that we need the `defaults` and `tasks` folder, with the `main.yml` in each of them

- `Check file` Here I want you to explain how to create the check code, with an actual example (put the code here in the docu). I want a description saying that check is task or a group of tasks that eventually generates a `true` or `false` output (`true -> check passed`, `false -> check failed`). And that this result must be passed to the `post-results` block, with the result in the `status` field. I want to be meticulous on this. Explain every bit, as you were writting a test for the 1st time
- `Metadata files`. Keep as it is. Maybe we could include a dummy metadata file content below the fields list, like in the check file